packages/web3-constants/evm/ethereum.json

```diff
@@ -17,7 +17,8 @@
         "Celo": "0x8e28F1d64ceD52b9A09aB1AA3071Aa3c05802d1F",
         "Fantom": "0xc119574d5fb333f5ac018658d4d8b5035e16bf39",
         "Aurora": "0xC119574D5Fb333F5AC018658D4d8b5035E16bf39",
-        "Aurora_Testnet": ""
+        "Aurora_Testnet": "",
+        "Conflux": ""
```

Comment:
Better to deploy it.

Reply:
Query to balance checker contract on conflux would cause infinite loading.